### DISCUSSION TITLE
Howto: custom sensor for status messages

### DISCUSSION THREAD

**Discussion by lordhelmet77** [Blackymas/NSPanel_HA_Blueprint#1211] [Q&A]
I like to use Sensor 03 on the Home screen to show custom status messages/events from multiple entities (in an unobtrusive way). Examples would be the laundry or dishwasher are finished or my mailbox sensor has been triggered.

The two challenges are:
1) Sensor 03 only can show one entity/sensor
2) The entities that trigger events have various types of output and I want something that's readable and fits the limited space for Sensor 03.

Is it possible to create a custom entity that:
- checks other entities for a predefined value
- and sets it own output to a predefined value?

I'm sure this can be done, but I'm not deep enough in HA to even ask the right questions I guess.

Any help is appreciated!

**Comment by edwardtfn** [comment]
Sorry, just saw that.
You can create a [template sensor](https://www.home-assistant.io/integrations/template/) in Home Assistant, which will get info from other sensors and use that to define it's own state, then assign this new sensor to your panel.
